src/importexport/musicxml/internal/musicxml/importmxmlpass2.cpp

```diff
@@ -4842,7 +4842,7 @@ void MusicXMLParserPass2::clef(const QString& partId, Measure* measure, const Fr
     if (strClefno != "") {
         clefno = _pass1.getMusicXmlPart(partId).staffNumberToIndex(strClefno.toInt());
     }
-    if (clefno < 0 || clefno >= part->nstaves()) {
+    if (clefno >= part->nstaves()) {
```

Comment:
This seems not safe either, but that problem has been introduced earlier by making `clefNo` a `size_t`.

Reply:
master uses the very same code

Reply:
But maybe that needs to get changed (back) too